Foundation/NSData.swift

```diff
@@ -217,7 +217,11 @@ open class NSData : NSObject, NSCopying, NSMutableCopying, NSSecureCoding {
     }
     
     open var bytes: UnsafeRawPointer {
-        return UnsafeRawPointer(CFDataGetBytePtr(_cfObject))
+        guard let bytes = CFDataGetBytePtr(_cfObject) else {
+            //This could occure on empty data being encoded.
+            return UnsafeRawPointer([])
+        }
+        return UnsafeRawPointer(bytes)
```

Comment:
Wouldn't changing signature of `bytes` break conformance to MacOS Foundation?.
Also rechecked `NSData` in MacOS Foundation. `.bytes` there is also UnsafeRawPointer, but returns nil if `.length` is 0. Missed this at first.
Would changing this to something like this work:
```swift
guard self.length > 0 else {
    return nil
}
return UnsafeRawPointer(CFDataGetBytePtr(_cfObject))
```
Or an issue with invalid address will persist?